--- ./src/test/java/org/apache/commons/csv/CSVPrinterTest.java.bak	2022-01-15 17:06:25.000000000 -0600
+++ ./src/test/java/org/apache/commons/csv/CSVPrinterTest.java	2022-01-15 17:22:44.000000000 -0600
@@ -1290,14 +1290,13 @@
 
     @Test
     public void testPrintRecordsWithResultSetOneRow() throws IOException, SQLException {
-        try (CSVPrinter csvPrinter = CSVFormat.MYSQL.printer()) {
+        CSVPrinter csvPrinter = CSVFormat.MYSQL.printer();
             final Value[] valueArray = new Value[0];
             final ValueArray valueArrayTwo = ValueArray.get(valueArray);
             try (ResultSet resultSet = valueArrayTwo.getResultSet()) {
                 csvPrinter.printRecords(resultSet);
                 assertEquals(0, resultSet.getRow());
             }
-        }
     }
 
     @Test
@@ -1316,13 +1315,12 @@
 
     @Test
     public void testPrintRecordsWithEmptyVector() throws IOException {
-        try (CSVPrinter csvPrinter = CSVFormat.POSTGRESQL_TEXT.printer()) {
+        CSVPrinter csvPrinter = CSVFormat.POSTGRESQL_TEXT.printer();
             final Vector<CSVFormatTest.EmptyEnum> vector = new Vector<>();
             final int expectedCapacity = 23;
             vector.setSize(expectedCapacity);
             csvPrinter.printRecords(vector);
             assertEquals(expectedCapacity, vector.capacity());
-        }
     }
 
     @Test
